app/Http/Controllers/MeetupController.php

```diff
@@ -71,6 +73,6 @@ public function sendEmailOnMeetupCreation(Meetup $meetup)
         Mail::send('emails.meetup-created', compact('user', 'meetupDetails'), function ($message) use ($user, $recipients) {
```

Comment:
**Code Quality - Critical**

You are storing the admin emails in `$recipients` variable which is an array. You are also storing the user's email in `$recipients[]` variable which is also an array. Should that be two different named variables where the first is an array and the second is a variable that just holds a string?